src/ViewFactory.cs

```diff
@@ -13,46 +11,122 @@ namespace TerminalGuiDesigner;
 /// sensible dimensions and content for dragging/configuring in
 /// the designer.
 /// </summary>
-public class ViewFactory
+public static class ViewFactory
```

Comment:
I normally don't like creating static classes as it can create limitations in future e.g. it prevents inheritence.

Reply:
Valid concern. Everything here was static and it's a factory, so it made sense at the time. But it can easily be non-static, since the methods are static.

It just wasn't necessary to instantiate it at any place it was used.

Easy enough to drop static on the class itself. Wouldn't require any other changes.